src/PhpSpreadsheet/Reader/Xml.php

```diff
@@ -167,6 +170,9 @@ public function listWorksheetInfo($pFilename)
             'SimpleXMLElement',
             Settings::getLibXmlLoaderOptions()
         );
+        if (!($xml instanceof \SimpleXMLElement)) {
```

Comment:
Should compare against `false` instead of checking types, so `if ($xml === false)`.

Error message should include error details from simplexml.